### ISSUE TITLE
Some unloaded tabs aren't distinguishable from loaded tabs

### ISSUE THREAD

**Issue by is-this-c** [brave/brave-browser#36720]
## Test plan
See https://github.com/brave/brave-core/pull/22852

## Description 

Some unloaded tabs aren't distinguishable from loaded tabs. They look exactly the same.

## Steps to Reproduce

   1. Unload tabs with urgent discard, or use a tab discard extension, or restore a session.

## Actual result:

These unloaded tabs aren't distinguishable from loaded tabs.

## Expected result:

In every case unloaded tabs should be distinguishable from loaded tabs.

## Reproduces how often: 

Always.

## Brave version (brave://version info)

Brave: 1.63.169 Chromium: 122.0.6261.111 (Official Build) (64-bit) 
Revision: e0a975cf46d4d026e184ca61788e1da6f0d76bc
OS: Windows 11 Version 23H2 (Build 22631.3155)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes.
- Can you reproduce this issue with the beta channel? Yes.
- Can you reproduce this issue with the nightly channel? Yes.

## Miscellaneous Information:

I went into brave://discards and proactive discard works. The tab is unloaded, and it becomes distinguishable with a ring.  If however I use urgent discard or a tab discard extension, they are able to unload a tab, but it'll continue to look like a loaded tab.

**Comment by rebron** [comment]
@is-this-c When you get a chance, can you post a screen shot? Also, is this the tab discard extension you're talking about `https://chromewebstore.google.com/detail/tab-discard-suspender/eackfaijnihopppechdgblecghggkome`

**Comment by is-this-c** [comment]
Yes, that's the extension. However, this issue applies to other tab discard extensions as well.

<img width="174" alt="Screenshot 2024-03-13 164911" src="https://github.com/brave/brave-browser/assets/87069698/fe3b7283-ba3f-4811-a36f-667f236dbd29">
</br>
</br>
<img width="180" alt="Screenshot 2024-03-13 170943" src="https://github.com/brave/brave-browser/assets/87069698/7556681b-4dce-4de3-b8c2-a8eecbcd1171">
</br>
</br>
<img width="179" alt="Screenshot 2024-03-13 172124" src="https://github.com/brave/brave-browser/assets/87069698/bf04ee82-aa22-49fa-9133-b61f632c9052">
</br>
</br>
<img width="180" alt="Screenshot 2024-03-13 172136" src="https://github.com/brave/brave-browser/assets/87069698/3d247c5e-758b-4711-94e1-0dba7f90ed6e">
</br>
</br>
<img width="826" alt="Screenshot 2024-03-13 164934" src="https://github.com/brave/brave-browser/assets/87069698/1fecc9ae-0395-4d0b-982e-7b6a26fdf955">

This issue does not exist with Edge.

<img width="741" alt="Screenshot 2024-03-13 172912" src="https://github.com/brave/brave-browser/assets/87069698/0ba6cd11-2ce0-4614-84a1-d148fb44a8bb">
</br>
Note: The third tab here is sleeped, which doesn't exist in Brave.

**Comment by is-this-c** [comment]
@rebron Have I provided enough info?

**Comment by is-this-c** [comment]
@rebron I took the liberty of making a PR last week. Is there anything else I need to do? It says that it is awaiting review. Does someone need to be assigned?

**Comment by is-this-c** [comment]
@rebron I have not heard anything in 2 months. Is my PR not going to be accepted?

**Comment by bsclifton** [comment]
@is-this-c sorry for the long turn-around; I'm going through the backlog of PRs and will try to find a reviewer. Thanks!

**Comment by is-this-c** [comment]
@bsclifton @rebron Thought I would ping after 5 months. Should I close this issue & PR? I thought maybe @bsclifton changed his mind after he approved it.

**Comment by bsclifton** [comment]
@is-this-c if you are able to rebase your original PR, I promise I can find a reviewer. I had trouble finding someone from @brave/chromium-src-reviewers - but then had forgot about this. Definitely my bad

Please let me know - would be great to have this. I approved because I pulled down the code, tried it, and it looked great to me

**Comment by is-this-c** [comment]
@bsclifton I rebased.

**Comment by simonhong** [comment]
Reopened as this fix is reverted - See https://github.com/brave/brave-browser/issues/36720
